samples/bluetooth/connection_event_trigger/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(NONE)
+
+FILE(GLOB app_sources src/*.c)
```

Comment:
no need for glob if you have one file. just put the file in the target_sources()